apps/root/src/pages/transfer/components/transfer-button/index.tsx

```diff
@@ -21,10 +24,19 @@ const TransferButton = ({ disableTransfer, onTransferClick }: TransferButtonProp
   const { network } = useTransferState();
   const actualCurrentNetwork = useCurrentNetwork();
   const activeWallet = useActiveWallet();
+  const wallets = useWallets();
   const walletService = useWalletService();
   const dispatch = useAppDispatch();
   const trackEvent = useTrackEvent();
   const isOnCorrectNetwork = actualCurrentNetwork.chainId === network;
+  const openConnectModal = useOpenConnectModal();
+  const { disconnect } = useDisconnect({
```

Comment:
This seems to be something we do everytime we open the connect modal, so probably should go inside the useOpenConnectModal hook